File: Java/Rana.java
public class Rana {
    private String color;
    private String sentido;
    
    public Rana(String color, String sentido){
        this.color = color;
        this.sentido = sentido;
    }

    public String getSentido(){
        return this.sentido;
    }

    public boolean vaPara(String sentido){
        return this.sentido == sentido;
    }

    @Override
    public String toString(){
        return this.sentido;
    }
}

File: Python/Rana.py
class Rana:
    def __init__(self, color:str, sentido:str):
        self.color = color
        self.sentido = sentido
        
    def get_sentido(self):
        return self.sentido
    
    def va_para(self, sentido:str):
        return self.sentido == sentido
    
    def __str__(self) -> str:
        return self.sentido